Makefile

```diff
@@ -48,7 +48,7 @@ install-docs-requirements:
 install-requirements: install-python-requirements install-node-requirements
 
 react-build:
-	cd owtf/webapp && yarn build
+	cd owtf/webapp && yarn run prod
```

Comment:
Is it possible to have command `make react-build` take arguments "dev" and "prod" so its flexible to use?

Reply:
I think there is a way we can do this using `make ENV="dev" react-build`.  But I think a cleaner solution would be to have two different make tags for this as we have now, though we can change the tags names to something `make web-dev` and `make web-prod`.

![image](https://user-images.githubusercontent.com/21292343/37214503-4fd33078-23db-11e8-996c-b73c73f780b6.png)

Reply:
I think the env approach is more cleaner since we don't need to do code duplication. So, it should look something like,

```
make OWTF_ENV={dev, prod}
```
and use this as `cd owtf/webapp && yarn run $(OWTF_ENV)`. However there should be default value if not env variable is defined.

Reply:
Yeah seems logical to me too. I will push the requested changes. :)